src/odemis/acq/align/autofocus.py

```diff
@@ -546,8 +546,7 @@ def estimateAutoFocusTime(detector, scanner=None, steps=MAX_STEPS_NUMBER):
     scanner (None or model.Emitter): In case of a SED this is the scanner used
     Estimates overlay procedure duration
     """
-    # Add 0.5s per step to account for the focus movement (very roughly approximated)
-    return steps * estimateAcquisitionTime(detector, scanner) + steps * 0.5
+    return steps * estimateAcquisitionTime(detector, scanner)
```

Comment:
That means the movement of the focus is instantaneous? That sounds a little bit optimistic. On the other hand, steps is MAX_STEPS_NUMBER, which is probably quite pessimistic. 
I can imagine it makes (by chance?) the estimation more realistic on the METEOR, but how many other cases did it worsen? 
I think you could try to make the focus movement more precise by:
* taking the `focus` component as argument
* try to guess the distance to move (on average)
* using the `estimateMoveDuration()`

I'd also suggest to use a smaller number than "steps". For instance "steps / 2", but it might be good to check typically how many steps are required. 
Hopefully this also gives you a "not too bad" estimate on the METEOR, and doesn't change too much the estimate on other systems.

Reply:
"I can imagine it makes (by chance?) the estimation more realistic on the METEOR, but how many other cases did it worsen?" The parameter that I deleted was added by me in a previous commit to improve the acquisition timer before. With that in mind, the other systems should work as usual ?